lib/rucio/client/baseclient.py

```diff
@@ -73,6 +73,14 @@ def choice(hosts):
     return secrets.choice(hosts)
 
 
+def _expand_path(p):
```

Comment:
Please type annotate this function and use a more descriptive variable name

Reply:
Done, though I changed more in this module due to it. The only pre-scriptive name I could think of was `path` and `os.path` was imported as `path`, which I already ran into while refactoring here.

So I removed this import and replaced usage of `path` with `os.path`. Let me now if this is ok or if I shoudl revert.